03-task-manager/starter/routes/tasks.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const router = express.Router()
+const { newTask, allTasks, getTask, updateTask, deleteTask } = require('../controllers/tasks')
+
+router.get('/', allTasks)
+router.post('/newTask', newTask)
```

Comment:
When working with REST APIs, we don't usually describe the operation using the route alone, but also the HTTP methods. Creation endpoints usually have the same route as the listing endpoints, but they use the POST method instead and that's what describes the operation.

So, instead of using the route `/netTask`, just leave it as it was and let the POST method be what defines that this endpoints creates new tasks.

```suggestion
router.post('/', newTask)
```

Reply:
Here's an article that talks about that and other best practices when working with REST:
https://blog.postman.com/rest-api-examples/#use-instead-of-verbs-in-endpoint-paths